runtime/util/hshelp.c

```diff
@@ -1663,7 +1665,7 @@ reresolveHotSwappedConstantPool(J9ConstantPool * ramConstantPool, J9VMThread * c
 }
 
 static void
-fixRAMConstantPoolForFastHCR(J9ConstantPool *ramConstantPool, J9HashTable *classHashTable, J9HashTable *methodHashTable)
+fixRAMConstantPoolForFastHCR(J9ConstantPool *ramConstantPool, J9HashTable *classHashTable, J9HashTable *methodHashTable, J9Class *objectClass)
```

Comment:
Won't this pass in a potentially stale object class ref?  I'm not clear on why passing it is better than fetching it from the VM here instead of in the caller?

Reply:
I pass it to ensure the original class pointer value (I want the stale one in order to compute indices into the method array).

Reply:
Ok, that makes sense